2022-09-10 15:24:35,695 INFO    MainThread:8398 [wandb_setup.py:_flush():71] setting env: {}
2022-09-10 15:24:35,695 WARNING MainThread:8398 [wandb_setup.py:_flush():71] could not find program at run.py
2022-09-10 15:24:35,695 INFO    MainThread:8398 [wandb_setup.py:_flush():71] setting login settings: {}
2022-09-10 15:24:35,695 INFO    MainThread:8398 [wandb_init.py:_log_setup():371] Logging user logs to /home/zzx/deepke/DeepKE/example/re/standard/logs/2022-09-10_15-24-34/wandb/run-20220910_152435-1d82chos/logs/debug.log
2022-09-10 15:24:35,695 INFO    MainThread:8398 [wandb_init.py:_log_setup():372] Logging internal logs to /home/zzx/deepke/DeepKE/example/re/standard/logs/2022-09-10_15-24-34/wandb/run-20220910_152435-1d82chos/logs/debug-internal.log
2022-09-10 15:24:35,695 INFO    MainThread:8398 [wandb_init.py:init():404] calling init triggers
2022-09-10 15:24:35,695 INFO    MainThread:8398 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-09-10 15:24:35,695 INFO    MainThread:8398 [wandb_init.py:init():449] starting backend
2022-09-10 15:24:35,695 INFO    MainThread:8398 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-09-10 15:24:35,705 INFO    MainThread:8398 [backend.py:ensure_launched():199] starting backend process...
2022-09-10 15:24:35,712 INFO    MainThread:8398 [backend.py:ensure_launched():204] started backend process with pid: 8439
2022-09-10 15:24:35,714 INFO    MainThread:8398 [wandb_init.py:init():458] backend started and connected
2022-09-10 15:24:35,715 INFO    MainThread:8398 [wandb_init.py:init():519] updated telemetry
2022-09-10 15:24:35,729 INFO    MainThread:8398 [wandb_init.py:init():549] communicating current version
2022-09-10 15:24:37,383 INFO    MainThread:8398 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.13.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-09-10 15:24:37,383 INFO    MainThread:8398 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-09-10 15:24:37,847 INFO    MainThread:8398 [wandb_init.py:init():592] starting run threads in backend
2022-09-10 15:24:39,459 INFO    MainThread:8398 [wandb_run.py:_console_start():1816] atexit reg
2022-09-10 15:24:39,459 INFO    MainThread:8398 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-09-10 15:24:39,460 INFO    MainThread:8398 [wandb_run.py:_redirect():1695] Redirecting console.
2022-09-10 15:24:39,461 INFO    MainThread:8398 [wandb_run.py:_redirect():1751] Redirects installed.
2022-09-10 15:24:39,461 INFO    MainThread:8398 [wandb_init.py:init():619] run started, returning control to user process
2022-09-10 15:24:39,623 INFO    MainThread:8398 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-09-10 15:24:39,623 INFO    MainThread:8398 [wandb_run.py:_restore():1758] restore
2022-09-10 15:24:42,473 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 669
}

2022-09-10 15:24:42,794 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 669
}

2022-09-10 15:24:42,895 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7061
}

2022-09-10 15:24:42,996 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7061
}

2022-09-10 15:24:43,097 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7061
}

2022-09-10 15:24:43,199 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7061
}

2022-09-10 15:24:43,300 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2527
  total_bytes: 7061
}

2022-09-10 15:24:43,401 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3626
  total_bytes: 7061
}

2022-09-10 15:24:43,503 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7061
  total_bytes: 7061
}

2022-09-10 15:24:43,604 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7061
  total_bytes: 7061
}

2022-09-10 15:24:43,705 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7061
  total_bytes: 7061
}

2022-09-10 15:24:43,807 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7061
  total_bytes: 7061
}

2022-09-10 15:24:43,908 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7061
  total_bytes: 7061
}

2022-09-10 15:24:44,009 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7061
  total_bytes: 7061
}

2022-09-10 15:24:44,594 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7061
  total_bytes: 7061
}

2022-09-10 15:24:45,275 INFO    MainThread:8398 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7061
  total_bytes: 7061
}
local_info {
}

2022-09-10 15:24:46,390 INFO    MainThread:8398 [wandb_run.py:_append_files():2184] logging synced files
